ui/app/src/Views/AutoComplete/Views.elm

```diff
@@ -0,0 +1,169 @@
+module Views.AutoComplete.Views exposing (view)
```

Comment:
I wouldn't nest more than 3 levels deep. I think it is fine right now. I only renamed `AutoComplete` to `GroupBar`.